### DISCUSSION TITLE
[@types/fs-extra] doe not export member 'EncodingOption'.

### DISCUSSION THREAD

**Discussion by GordonHPepBoys** [DefinitelyTyped/DefinitelyTyped#69275] [Issues with a @types package] [answered]
All, I have this following error while trying to build project in aws

```
node_modules/@types/fs-extra/index.d.ts(917,48): error TS2694: Namespace '"fs"' has no exported member 'EncodingOption'.
```

environment :
@types/fs-extra: "^11.0.4",
 typescript : "5.3.3",
node.js: 20.1.1
Please advise. 
Thanks, Gordon

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Please upgrade to latest version of `@types/node` and try again.

**Reply by GordonHPepBoys** [reply]
It worked. But I have another issue with another type.

**Reply by ghost** [reply]
What is the issue?
